ts/screens/messages/MessagesScreen.tsx

```diff
@@ -32,10 +33,24 @@ export type OwnProps = Readonly<{
 }>;
 
 export type IMessageDetails = Readonly<{
-  item: Readonly<MessageWithContentPO>;
+  item: Readonly<IMessageWithPaymentData>;
   index: number;
 }>;
 
+export type IPaymentData = Readonly<{
+  amount: number;
+  notice_number: string;
+}>;
+
+export type IMessageWithPaymentData = Readonly<{
+  id: string;
+  created_at: string;
```

Comment:
`Date`?

Reply:
you may also consider a DateFromString (if it's tolerable to decode it with io-ts):
https://github.com/teamdigitale/italia-ts-commons/blob/master/src/dates.ts#L5

Reply:
I receive the date as string from MessageWithContentPO (ts/types)
/**
 * A plain object representation of a MessageWithContent useful to avoid problems with the redux store.
 * The create_at date object is transformed in a string.
 */
const MessageWithContentPO = t.exact(
  t.intersection(
    [
      t.type({
        ...MessageWithContent.type.types[0].props,
        created_at: t.string
      }),
      MessageWithContent.type.types[1]
    ],
    "MessageWithContentPO"
  )
);
and after the date it's managed from the function convertDateToWordDistance(date: Date, lastDayLabel: string) => convertDateToWordDistance(new Date(created_at), ...)

Reply:
To avoid problems with redux-persist, that can't store complex object like Date, when fetching the messages from the backend every MessageWithContent is converted to MessageWithContentPO.

Reply:
it would make more sense to keep `MessageWithContent` as the runtime representation and convert to `MessageWithContentPO` before persisting (redux-persist is an ORM)